cmd/mo-service/config.go

```diff
@@ -184,6 +185,10 @@ func (c *Config) createFileService(defaultName string, perfCounter *perfcounter.
 		services = append(services, service)
 		perfCounterInfos = append(perfCounterInfos, counter)
 
+		// Create "Log Exporter" for this PerfCounter
```

Comment:
@reusee Right now, I am registering this global counter to the `statsRegistry` inside the `for loop`. It seems like this `counter.fsName` can be duplicated (eg, LOCAL). Could you suggest me a counter which has a unique familyName?